tests/integration/scale/test_scale.py

```diff
@@ -160,3 +161,51 @@ def test_scale_fail(flow_with_runtime, pod_params, mocker):
         for replica_id in r2.docs.get_attributes('tags__replica_id'):
             replica2_ids.add(replica_id)
     assert replica1_ids == replica2_ids == {0, 1}
+
+
+@pytest.mark.parametrize(
+    'pod_params',
+    [
+        (2, 3, 1),
+    ],
+    indirect=True,
+)
+@pytest.mark.parametrize('protocol', ['websocket', 'grpc', 'http'])
+def test_scale_with_client(flow_with_runtime, pod_params, protocol):
+    num_replicas, scale_to, shards = pod_params
+    with flow_with_runtime as f:
+        f.protocol = protocol
+        client = Client(
```

Comment:
Why is this relevant? This is equivalent to using the `flow`. The relevant thing would be to have the clients send requests in parallel while the `scale` is happening